avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -8,10 +8,12 @@
 com1Exe = ./com1DFA.exe -files files/AK_Attributes/
 # True if full output shall be printed to terminal
 fullOut = False
-# True if entrainment areas shall NOT be included
-noEntrainment = False
-# True if resistance areas shall NOT be included
-noResistance = False
+
+#++++++++++++++++ simulation type
+# list of simulations that shall be performed (null, ent, res, entres, available (use all available input data))
+simTypeList = null|available
```

Comment:
should be only `available`, since `available` contains `null`...

Reply:
correct, set it to available